src/wacks4shop/wack.py

```diff
@@ -20,6 +22,16 @@
 logger = logging.getLogger("wacks4shop")
 
 
+def log_waterline(db: Wackabase, timestamp: Optional[str]):
```

Comment:
why not make the timestamp required and move the error log outside this function?